### PR TITLE
html-movie-seat-booking

### PR DESCRIPTION
**Pull request by 7SergSmith7**
##HTML movie seat booking

[Demo](https://7sergsmith7.github.io/HTML-movie-seat-booking/)
[Code](https://github.com/kottans/frontend-2021-classroom/tree/7SergSmith7/html-movie-seat-booking/submissions/7SergSmith7/html-movie-seat-booking)

Please, review!

### CODE REVIEW

File: submissions/7SergSmith7/html-movie-seat-booking/index.html [hunk 1]
```diff
@@ -0,0 +1,791 @@
...
+                <div class="row__seats row-seats">
+                  <span class="row-seats__number">8</span>
+                  <span class="row-seats__title">Row</span>
+                </div>
+                <div class="row__seats seat-cost">
+                  <span class="seat-cost__number">$5</span>
+                  <span class="seat-cost__title">Cost</span>
+                </div>
+              </div>
+              <span class="row__delete-icon">X</span>
```
Comment (A-Ostrovnyy) [inline submissions/7SergSmith7/html-movie-seat-booking/index.html]: Is this an interactive element?

File: submissions/7SergSmith7/html-movie-seat-booking/index.html [hunk 2]
```diff
@@ -0,0 +1,791 @@
...
+        <img
+          class="main__film-description__film-image"
+          src="soul_image.png"
+          alt="soul_image"
+        />
+        <h2 class="main__film-description__film-title">soul</h2>
+        <span class="main__film-description__film-year">2021</span>
+        <div class="main__film-description__selected-seats-info">
+          <h3 class="selected-seats-info__title">Selected Seats</h3>
+          <div class="selected-seats-info__seats-list">
```
Comment (A-Ostrovnyy) [inline submissions/7SergSmith7/html-movie-seat-booking/index.html:34]: Content in this block is a structured set of data and maybe better to use `table` here?

File: submissions/7SergSmith7/html-movie-seat-booking/index.html [hunk 3]
```diff
@@ -0,0 +1,791 @@
...
+            >
+            <span class="selected-seats-info__total-cost-block__total-cost"
+              >$10</span
+            >
+          </div>
+          <a href="#" class="selected-seats-info__checkout-btn">checkout</a>
+        </div>
+      </section>
+      <section class="main__screen-and-seats-block">
+        <nav class="main__screen-and-seats-block__navigation">
```
Comment (A-Ostrovnyy) [inline submissions/7SergSmith7/html-movie-seat-booking/index.html]: Logically this block for choosing movie session parameters, not for navigation.

File: submissions/7SergSmith7/html-movie-seat-booking/index.html [hunk 4]
```diff
@@ -0,0 +1,791 @@
...
+          <div class="main__screen-and-seats-block__navigation__date">
+            <ul class="navigation__date__list">
+              <li class="date__list__item navigation-arrow"><</li>
```
Comment (A-Ostrovnyy) [inline submissions/7SergSmith7/html-movie-seat-booking/index.html]: You use list item `li` for an interactive element, don't do that in the future)

File: submissions/7SergSmith7/html-movie-seat-booking/index.html [hunk 5]
```diff
@@ -0,0 +1,791 @@
...
+              <li class="date__list__item selected-date">02 February</li>
```
Comment (A-Ostrovnyy) [inline submissions/7SergSmith7/html-movie-seat-booking/index.html]: For date/time use html tag `time` with filled `datetime` attribute.

File: submissions/7SergSmith7/html-movie-seat-booking/index.html [hunk 6]
```diff
@@ -0,0 +1,791 @@
...
+          />
+        </div>
+        <div class="main__screen-and-seats-block__seats-list">
+          <div class="seats-list__row">
+            <label
+              class="seats-list__row__item empty-place"
+              for="seat-1-1"
+            ></label>
+
+            <label
```
Comment (A-Ostrovnyy) [inline submissions/7SergSmith7/html-movie-seat-booking/index.html:211]: Add attribute `value` to input.

File: submissions/7SergSmith7/html-movie-seat-booking/index.html [hunk 7]
```diff
@@ -0,0 +1,791 @@
...
+            </ul>
+          </div>
+          <div class="main__screen-and-seats-block__navigation__time">
+            <ol class="navigation__time__list">
+              <li class="time__list__item">10:00</li>
+              <li class="time__list__item">11:20</li>
+              <li class="time__list__item selected-time">12:00</li>
+              <li class="time__list__item">16:00</li>
+              <li class="time__list__item">17:20</li>
+              <li class="time__list__item">19:00</li>
```
Comment (A-Ostrovnyy) [inline submissions/7SergSmith7/html-movie-seat-booking/index.html]: Logically these elements are intended for choosing a movie session time, which means you need to choose interactive elements for this.

File: submissions/7SergSmith7/html-movie-seat-booking/index.html [hunk 8]
```diff
@@ -0,0 +1,791 @@
...
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <title>Movie Seat Booking</title>
+  </head>
+  <body>
+    <header>
+      <h1 class="header__title">Movie Seat Booking</h1>
```
Comment (A-Ostrovnyy) [inline submissions/7SergSmith7/html-movie-seat-booking/index.html]: If you use BEM methodology. BEM element can't be outside BEM block. In your case, you use `.header_title` as BEM element without BEM block `.header`.
Also, no need to inherit all parent's hierarchy in class name.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comments below!

File: submissions/7SergSmith7/html-movie-seat-booking/index.html (hunk 1 above)
Comment (7SergSmith7) [reply submissions/7SergSmith7/html-movie-seat-booking/index.html]: I planned so)

File: submissions/7SergSmith7/html-movie-seat-booking/index.html (hunk 2 above)
Comment (7SergSmith7) [reply submissions/7SergSmith7/html-movie-seat-booking/index.html:34]: Is this a critical moment?

File: submissions/7SergSmith7/html-movie-seat-booking/index.html (hunk 4 above)
Comment (7SergSmith7) [reply submissions/7SergSmith7/html-movie-seat-booking/index.html]: What is the best tag to use in this case?

File: submissions/7SergSmith7/html-movie-seat-booking/index.html (hunk 7 above)
Comment (7SergSmith7) [reply submissions/7SergSmith7/html-movie-seat-booking/index.html]: Do I need to use a button or radio?

File: submissions/7SergSmith7/html-movie-seat-booking/index.html (hunk 2 above)
Comment (A-Ostrovnyy) [reply submissions/7SergSmith7/html-movie-seat-booking/index.html:34]: Nope, l just asked to know your opinion about that)

Comment (7SergSmith7) [reply submissions/7SergSmith7/html-movie-seat-booking/index.html:34]: I will keep in mind)

File: submissions/7SergSmith7/html-movie-seat-booking/index.html (hunk 3 above)
Comment (7SergSmith7) [reply submissions/7SergSmith7/html-movie-seat-booking/index.html]: done

File: submissions/7SergSmith7/html-movie-seat-booking/index.html (hunk 5 above)
Comment (7SergSmith7) [reply submissions/7SergSmith7/html-movie-seat-booking/index.html]: done

File: submissions/7SergSmith7/html-movie-seat-booking/index.html (hunk 6 above)
Comment (7SergSmith7) [reply submissions/7SergSmith7/html-movie-seat-booking/index.html:211]: done

File: submissions/7SergSmith7/html-movie-seat-booking/index.html (hunk 7 above)
Comment (7SergSmith7) [reply submissions/7SergSmith7/html-movie-seat-booking/index.html]: done

File: submissions/7SergSmith7/html-movie-seat-booking/index.html (hunk 8 above)
Comment (7SergSmith7) [reply submissions/7SergSmith7/html-movie-seat-booking/index.html]: done

File: submissions/7SergSmith7/html-movie-seat-booking/index.html (hunk 4 above)
Comment (A-Ostrovnyy) [reply submissions/7SergSmith7/html-movie-seat-booking/index.html]: Move interactive element outside `ul`

File: submissions/7SergSmith7/html-movie-seat-booking/index.html (hunk 1 above)
Comment (A-Ostrovnyy) [reply submissions/7SergSmith7/html-movie-seat-booking/index.html]: For interactive elements use corresponding interactive HTML tags.

File: submissions/7SergSmith7/html-movie-seat-booking/index.html (hunk 5 above)
Comment (A-Ostrovnyy) [reply submissions/7SergSmith7/html-movie-seat-booking/index.html]: Not fixed.

File: submissions/7SergSmith7/html-movie-seat-booking/index.html (hunk 3 above)
Comment (A-Ostrovnyy) [reply submissions/7SergSmith7/html-movie-seat-booking/index.html]: I mean don't use tag `nav`.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work.
A lot of issues need to be fixed, don't be hesitate to ask if you don't understand something.
Also, remove files not for review(images, etc)

File: submissions/7SergSmith7/html-movie-seat-booking/index.html (hunk 1 above)
Comment (A-Ostrovnyy) [reply submissions/7SergSmith7/html-movie-seat-booking/index.html]: Use tag `<button>` for cases like this.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
Fo future: check and finish all issues before requesting review)

File: submissions/7SergSmith7/html-movie-seat-booking/index.html (hunk 1 above)
Comment (7SergSmith7) [reply submissions/7SergSmith7/html-movie-seat-booking/index.html]: sorry...
Fixed

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**